packages/bappo-components/src/primitives/Switch/Switch.web/Switch.tsx

```diff
@@ -0,0 +1,155 @@
+import * as React from 'react';
+import styled from 'styled-components';
+
+import { InputHandle } from '../../../input-handle';
+import { DivViewBase } from '../../../internals/web/ViewBase';
+import { SwitchProps } from '../types';
+
+type Props = SwitchProps & {
+  // Will be removed
+  className?: string;
+};
+
+const Switch = React.forwardRef(
+  (
+    {
+      accessibilityLabel,
+      disabled = false,
+      onBlur,
+      onFocus,
+      onValueChange,
+      style,
+      testID,
+      value = false,
+      className,
+    }: Props,
+    ref: React.Ref<InputHandle>,
+  ) => {
+    const containerRef = React.useRef<HTMLDivElement>(null);
+
+    React.useImperativeHandle(ref, () => ({
+      focus: () => {
+        if (containerRef && containerRef.current) containerRef.current.focus();
+      },
+      blur: () => {
+        if (containerRef && containerRef.current) containerRef.current.blur();
+      },
+    }));
+
+    const _onBlur = (event: React.FocusEvent<HTMLDivElement>) => {
+      onBlur &&
+        onBlur({
+          nativeEvent: {
+            value,
+          },
+        });
```

Comment:
Just so you know, you can use this shorthand:
```suggestion
      onBlur?.({
        nativeEvent: {
          value,
        },
      });
```